src/coreComponents/fileIO/timeHistory/PackCollection.cpp

```diff
@@ -200,55 +240,54 @@ void PackCollection::buildMetaCollectors( )
     {
       coordField = ElementSubRegionBase::viewKeyStruct::elementCenterString();
     }
-    else
-    {
-      GEOSX_ERROR( "Data collection for \"" << m_objectPath << "\" is unimplemented." );
-    }
+//    else
+//    {
+//      GEOSX_ERROR( "Data collection for \"" << m_objectPath << "\" is unimplemented." );
+//    }
 
-    string metaName( "coordinates" );
-    std::unique_ptr< PackCollection > coordCollector = std::make_unique< PackCollection >( metaName, this );
-    coordCollector->getWrapper< string >( PackCollection::viewKeysStruct::objectPathString() ).reference() = m_objectPath;
-    coordCollector->getWrapper< string >( PackCollection::viewKeysStruct::fieldNameString() ).reference() = string( coordField );
-    coordCollector->getWrapper< string_array >( PackCollection::viewKeysStruct::setNamesString() ).reference() = m_setNames;
-    coordCollector->getWrapper< localIndex >( PackCollection::viewKeysStruct::onlyOnSetChangeString() ).reference() = 1;
-    // don't recursively keep creating coordinate metacollectors
-    coordCollector->disableCoordCollection( );
-    m_metaCollectors.push_back( std::move( coordCollector ) );
+    string metaName( "coordinates" ); // FIXME Name?
```

Comment:
@francoishamon I'm not very qualified about this name.
Do you have any advice?
I'd go for `collect`, `collected` or `collector`?

Reply:
I'm not completely aware of all of this, but I've been trying the cases of the related issues, and I could not spot the `coordinates` keyword in the `hdf5` files. I do not have the whole flow in my mind...

Reply:
It looks like I was indeed misremembering, since e.g. the async case from #1851 produces the following datasets:
```
fluid_phaseDensity       Dataset {100/Inf, 1827/Inf, 2}
fluid_phaseDensity\ Time Dataset {100/Inf, 1/Inf}
fluid_phaseDensity\ elementCenter Dataset {100/Inf, 1827/Inf, 3}
```
Where the `... elementCenter ...` dataset is what is collected by the `coordinates` collector, so I guess really this name is fairly arbitrary.

Reply:
OK; most probably some `Group` constraint that must have a `name`.
I'll go for flat `metaCollector` then.